Add workflow for updating rules

bensonce opened the pull request:
Added a workflow that runs every two weeks to update the list of managed rules. If changes are found then a PR is opened and reviewers are automatically assigned. If for some reason no one reviews the PR before the next workflow run, the workflow should update the existing PR with changes again so multiple PRs aren't cluttering the repo.

The first PR using this workflow can be found here: https://github.com/niaid/terraform-aws-managed-config-rules/pull/31

duraikkannuv2 reviewed (commented):
Looks good. Thanks for adding this. We need to also update the conformance pack by running  the script `scripts/generate-rule-pack-info.rb`

bensonce commented:
> Looks good. Thanks for adding this. We need to also update the conformance pack by running the script `scripts/generate-rule-pack-info.rb`

Good idea! I've added a new workflow for that here too. This one is manually triggered but also opens a new PR with any changes. I rewrote the script in Python and reorganized the scripts to all be in the same `scripts` directory. The new `rule_pack_info_generator.py` script was tested locally and behaves the same as the old Ruby script. You can test it locally yourself to see the changes, too.

duraikkannuv2 reviewed (approved):
Looks good to me!
